build.gradle

```diff
@@ -91,42 +89,35 @@ task unzipCloudResourceSchema(type: Copy) {
     from zipTree(configurations.cloudResourceSchema.singleFile).matching {
         include 'cloud_resources_uid.yaml'
     }
-    into "$openapiTargetDir"
```

Comment:
should we remove the other referenes to 'openapi' in the comments and variables on lines 73-80? Or leave them, since we're still usin gthe openapi specification, just using swagger codegen to create the code? WDYT?

Reply:
I think OpenAPi is still the correct name. We just switch the the way to generate openapi java code from openApiGenator code to Swager-CodeGen